src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, Countable {
+    private String color;
+    private String name;
+
+    public Figure() {
+    }
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public Figure(String color, String name) {
+        this.color = color;
+        this.name = name;
+    }
+
+    public void draw() {

```

Comment:
The `draw()` method should not print out a message directly. Consider using a `toString()` method to return the string representation of the object, and then print that message outside the class.